packages/node_modules/@webex/internal-plugin-search/src/search.js

```diff
@@ -59,7 +59,14 @@ const Search = WebexPlugin.extend({
           searchEncryptionKeyUrl: this.webex.internal.device.searchEncryptionKeyUrl
         })
       }))
-      .then((res) => get(res, 'body.activities.items') || []);
+      .then((res) => {
+        const resActivities = get(res, 'body.activities.items', []);
```

Comment:
Nice, I didn't know about the default value param from `get`!

Reply:
Phat helped me find that I didn't know either!